opal/mca/allocator/bucket/allocator_bucket_alloc.c

```diff
@@ -89,6 +94,13 @@ void *mca_allocator_bucket_alloc(mca_allocator_base_module_t *mem, size_t size)
         bucket_size <<= 1;
     }
 
+    if( bucket_num > max_bucket_idx ) {
+        size_t sz_bucket = MCA_ALLOCATOR_BUCKET_1_SIZE;
+        fprintf (stderr, "ERROR: Requested buffer size %zu exceeds limit of %zu\n", size, sz_bucket << max_bucket_idx);
+        fprintf (stderr, "Consider setting \"allocator_bucket_num_buckets\" to %d\n", bucket_num + 1);
```

Comment:
These `fprintf()`s should probably be `opal_show_help()`s.

Reply:
done - fprintfs replaced with opal_show_help